cli/src/lib.rs

```diff
@@ -215,12 +215,10 @@ impl Iroha {
             config.sumeragi.trusted_peers.clone(),
         );
 
-        let kura = Kura::new(&config.kura)?;
+        let (kura, block_count) = Kura::new(&config.kura)?;
         let kura_thread_handler = Kura::start(Arc::clone(&kura));
```

Comment:
can this also be inlined?

Reply:
yeah, but my thinking was that for testing purposes we don't need starting write to disk loop